src/main/scala/com/databricks/labs/overwatch/pipeline/Pipeline.scala

```diff
@@ -425,6 +427,33 @@ class Pipeline(
     Helpers.getDatesGlob(startDate.toLocalDate, untilTime.asLocalDateTime.plusDays(1).toLocalDate)
   }
 
+  //Get the dataframe from either tempDirectory or from the ResponseArray
+  private def getApiData(apiTempPath: Option[String]): DataFrame ={
+     //Get the data from either tempDirectory or from the ResponseArray
+     val tmpClusterEventsSuccessPath = s"${config.tempWorkingDir}/${apiTempPath.get}/success" + config.runID
+    val tmpClusterEventsErrorPath = s"${config.tempWorkingDir}/${apiTempPath.get}/error" + config.runID
+    null
+  }
```

Comment:
is this right? it shouldn't be /error + configr.runId -- you'll need a slash or underscore or something here, no?

Reply:
added _